docs/Clojure.md

```diff
@@ -0,0 +1,125 @@
+# How to use Minio SDK for java with Clojure
+
+In this receipe we will learn how to use `minio-java` with Clojure.
+
+## Prerequisites
+Install `Clojure` and `Leiningen`.
+
+Check by running -
+```
+lein --help
+```
+
+## Steps for setup
+Create a new Clojure project using Leiningen.
+```
+lein new <templateName> <projectName>
+```
+
+```
+lein new app clojproject
+```
+
+Navigate to `project.clj` look at `:dependencies` section, listing what project requires in order to load, compile and run.
+```
+lein deps
+```
+Run this command to install list of dependencies to your project.
+
+Use Leiningen to start Clojure now - 
+```
+lein repl
+```
+Or you can run the project using command - `lein run`.
+
+Test the project by running `(-main)`.
+```
+user=> (-main)
+Hello, World!
+nil
+```
+
+## Getting started with minio-java
+
+First, download minio-java.
+```
+git clone git@github.com:minio/minio-java.git
+```
+
+Compile a jar by running this command in minio-java directory - 
+```
+./gradlew build
+```
```

Comment:
Is it mandatory to build `minio-java` from source? I think we can point to http://repo1.maven.org/maven2/io/minio/minio/ for users to simply download the jar. For advanced users (who want to build from source) we can add this link https://github.com/minio/minio-java/blob/master/CONTRIBUTING.md#contributors-guide